src/snc/agents/hedgehog/strategic_idling/strategic_idling_foresight.py

```diff
@@ -58,7 +59,7 @@ def __init__(self,
         self._initialise_fluid_policy(policy_object)
         self._compute_num_roll_out_steps()
 
-    def get_allowed_idling_directions(self, state: StateSpace) -> StrategicIdlingOutput:
+    def get_allowed_idling_directions(self, state: StateSpace, safety_stocks_vec) -> StrategicIdlingOutput:
```

Comment:
This seems out of nowhere here and difficult to know when it will be used. Can we do anything about that? Happy to discuss...